index.js

```diff
@@ -0,0 +1,101 @@
+const { PrismaClient } = require('@prisma/client')
+const { v4: uuidv4 } = require('uuid')
+
+const prisma = new PrismaClient()
+
+// retryTxn wraps SQL operations in an explicit transaction.
+// If the transaction fails, the function retries the operation, with exponential backoff.
+async function retryTxn (n, max, operation) {
```

Comment:
We have a similar feature over [here](https://www.prisma.io/docs/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) (without retry logic).